unit-tests/run-unit-tests.py

```diff
@@ -404,13 +407,12 @@ def test_wrapper( test, configuration = None, repetition = 1 ):
         devices.query()
         devices.map_unknown_ports()
         #
-        # Under Travis, we'll have no devices and no acroname
+        # Under a vendor's CI system, we'll have no devices and no acroname
```

Comment:
A vendor's CI system?
For a developer manually running LibCI there'll be no (or maybe one or more) device but no acroname.

Reply:
It said Travis which is no longer relevant.
`# Under a development environment, we'll may not have devices and no acroname`?

Reply:
Done